onboarding/src/Components/FormsEdit/FormsSectionsPreview.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import parse from "html-react-parser";
+
+function FormsSectionsPreview({ section }) {
+  const answersList = section.data.map((answer, i) => {
+    if (section.type === "osa" || section.type === "msa") {
+      return (
+        <tr key={ `answ-${answer.id}` }>
```

Comment:
`Warning: Encountered two children with the same key, 'answ-undefined'. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.` :open_mouth:  :/